[pt][ja Fix file path for concepts/instrumentation/zero-code/index.md

vitorvasc opened the pull request:
<!--
Please provide a meaningful description of what your changes will do. Bonus points for including links to related issues, other PRs, or technical references.

A detailed description helps reviewers understand the context of your change and reduces
the time needed to review.

Please make sure to follow the [contribution guidelines](https://opentelemetry.io/docs/contributing/).

We run checks on all PRs, to ensure that your contribution follows our [style guide](https://opentelemetry.io/docs/contributing/style-guide/).

Many requirements of that guide can be enforced by running `npm run fix:all` locally and committing the changes.

Note, that although those checks need to be green before we can merge your PR, do not
worry about their status after submitting your PR. We will provide you with guidance, when and how to
address them.

Similarly, do not worry about your PR being out-of-date with the base branch and do not continuously
update or rebase your branch. We will let you know when it is necessary.

If you have any questions, feel free to ask.
-->

ymotongpoo commented:
Does this change affect the link in this page? https://opentelemetry.io/docs/getting-started/ops/

vitorvasc commented:
> Does this change affect the link in this page? https://opentelemetry.io/docs/getting-started/ops/

Hi @ymotongpoo. I don't see any issue for this page, as the URLs remains the same.

* https://deploy-preview-5882--opentelemetry.netlify.app/pt/docs/concepts/instrumentation/zero-code/
* https://deploy-preview-5882--opentelemetry.netlify.app/ja/docs/concepts/instrumentation/zero-code/

ymotongpoo commented:
Thank you, @vitorvasc ! Then it looks great to me!

chalin reviewed (changes requested):
Hi @vitorvasc : thanks for the PR. Here are a few comments about how we handle changes in locale-specific docs:

1. We don't encourage changes that span locales. Please resubmit this for `pt` only and either submit a separate entry for `ja` pages or let `ja` approvers handle the update.
2. Any change to any non-`en` page much update the `default_lang_commit` commit hash, while ensuring that _all_ changes introduced up until that new commit hash have been incorporated into this PR.

For all the details about handling non-English page drift, see [Keeping track of localized-page drift](https://opentelemetry.io/docs/contributing/localization/#track-changes).

vitorvasc commented:
@chalin thanks for the heads up. Just updated the PR by replicating the changes on the `en` version -- actually, didn't notice that it was actually a change beyond the file being renamed.

Also, removed the `ja` translation from this pull request, so, this should cover only the `pt` version.

By the way, should we translate the content for the .svg file? cc @maryliag

EzzioMoreira reviewed (approved):
LGTM!!
